clarity/src/vm/database/clarity_db.rs

```diff
@@ -1205,6 +1205,23 @@ impl<'a> ClarityDatabase<'a> {
         )
     }
 
+    /// Like fetch_entry_unknown_descriptor, except that it expects
+    ///  to receive a value (i.e., it expects the state to exist)
+    /// This should only ever be invoked outside of Clarity, e.g. by the VM
+    ///  when loading state from a known-contract
```

Comment:
This is really only ever used with `.pox-2`, right?  Also, is there a way to programmatically enforce this property?

Reply:
I'm not sure if there's a way to enforce this programmatically.

Ultimately, this could also be useful for other contracts that the Stacks chainstate fetches expectant data from.

Reply:
Removed this method as it was only necessary in the previous implementation.